addon/templates/components/polaris-resource-list/bulk-actions.hbs

```diff
@@ -0,0 +1,139 @@
+        {{#polaris-popover
+          onClose=(action "toggleSmallScreenPopover")
+          as |popover|
+        }}
+          {{#popover.activator}}
+            {{polaris-resource-list/bulk-actions/bulk-action-button
+              content="Actions"
+              onAction=(action "toggleSmallScreenPopover")
+            }}
+          {{/popover.activator}}
+
+          {{#popover.content}}
+            {{polaris-action-list
+              items=promotedActions
+              sections=actionSections
+              onActionAnyItem=(action "toggleSmallScreenPopover")
+            }}
+          {{/popover.content}}
+        {{/polaris-popover}}
+      </div>
+    {{/if}}
+
+    <button
+      class="Polaris-ResourceList-BulkActions__Button Polaris-ResourceList-BulkActions__Button--cancel"
+      onclick={{action "setSelectMode" false}}
+    >
+      Cancel
+    </button>
+  </div>
+
+  {{#if (or paginatedSelectAllAction (and paginatedSelectAllText paginatedSelectAllAction))}}
+    <div class="Polaris-ResourceList-BulkActions__PaginatedSelectAll">
+      {{#if (and paginatedSelectAllText paginatedSelectAllAction)}}
+        <span aria-live="polite">{{paginatedSelectAllText}}</span>
+      {{else}}
+        {{paginatedSelectAllText}}
+      {{/if}}
+
+      {{#if paginatedSelectAllAction}}
+        {{polaris-button
+          plain=true
+          disabled=disabled
+          text=paginatedSelectAllAction.content
+          onClick=(action paginatedSelectAllAction.onAction)
+        }}
+      {{/if}}
+    </div>
+  {{/if}}
+</div>
+
+<div
+  class="Polaris-ResourceList-BulkActions__Group
+  Polaris-ResourceList-BulkActions__Group--largeScreen
+  {{if measuring 'Polaris-ResourceList-BulkActions__Group--measuring'}}
+  {{if selectMode 'Polaris-ResourceList-BulkActions__Group--entered' 'Polaris-ResourceList-BulkActions__Group--exited'}}"
+>
+  <div class="Polaris-ResourceList-BulkActions__ButtonGroup">
+    {{polaris-resource-list/checkable-button
+      accessibilityLabel=accessibilityLabel
+      label=label
+      selected=selected
+      selectMode=selectMode
+      measuring=measuring
+      disabled=disabled
+      onToggleAll=(action onToggleAll)
+    }}
+
+    {{#each promotedActionsToRender as |promotedAction|}}
+      {{polaris-resource-list/bulk-actions/bulk-action-button
+        content=promotedAction.content
+        onAction=(action promotedAction.onAction)
+        handleMeasurement=(action "handleMeasurement")
+      }}
+    {{/each}}
+
+    {{#if shouldRenderActionsPopover}}
+      <div class="Polaris-ResourceList-BulkActions__Popover">
+        {{!-- TODO: set `active` to largeScreenPopoverVisible once implemented --}}
+        {{#polaris-popover
+          onClose=(action "toggleLargeScreenPopover")
+          as |popover|
+        }}
+          {{#popover.activator}}
+            {{polaris-resource-list/bulk-actions/bulk-action-button
+              content=activatorLabel
+              disabled=disabled
```

Comment:
Check the behaviour of the popover here when `disabled` is truthy - I found the other day that despite setting the popover activator button to be disabled, the activator itself still worked, so the popover could still be triggered even though the button itself was disabled 😬 I worked around it by doing this:

```hbs
{{#component (if disabled
  (component "wrapper-element" tagName="")
  popover.activator
)}}
  {{polaris-button
    ...
    disabled=disabled
    ...
  }}
{{/component}}
```

to swap out the popover activator when the button's disabled, but maybe there's a better way? 🤔

Reply:
Yeah it's pretty icky 😬